src/types.ts

```diff
@@ -155,5 +159,7 @@ export type LegacyPartialDeploymentHistory = {
     offset: number
     limit: number
     moreData: boolean
+    next?: string
```

Comment:
Actually, since we deleted `/history`, we can delete `LegacyDeploymentEvent`, `LegacyDeploymentHistory` and `LegacyPartialDeploymentHistory`